drivers/dma/Kconfig.dw_common

```diff
@@ -38,3 +38,9 @@ config DMA_DW_HOST_MASK
 	help
 	  Some instances of the DesignWare DMAC require a mask applied to source/destination
 	  addresses to signifiy the memory space the address is in.
+
+config DMA_DW_CHANNEL_COUNT
```

Comment:
This is done through the common device tree binding, dts/bindings/dma/dma-controller.yaml instead and should not be needed as a Kconfig

Reply:
This Config is used to limit count of struct objects in an array. So can't use dma-channel dts parameter for this.
And channel count varies between platforms and not instances. I think this config is needed.